packages/shared/src/components/notifications/EnableNotification.tsx

```diff
@@ -135,17 +137,19 @@ function EnableNotification({
       <div className="flex mt-4 align-center">
         {!hasEnabled && (
           <Button
-            buttonSize={ButtonSize.Small}
-            className="mr-4 min-w-[7rem] btn-primary-cabbage"
+            size={ButtonSize.Small}
+            variant={ButtonVariant.Primary}
+            color={ButtonColor.Cabbage}
+            className="mr-4 min-w-[7rem]"
```

Comment:
This might have some use-case but was the `min-w` needed here?

Reply:
Looking at the visuals, it seems like the size of the button is more than the specified min width in all of the viewports (large, tablet, phone). So removing the min-w makes sense 👍